manifests/0000_90_console-operator_02_servicemonitor.yaml

```diff
@@ -0,0 +1,51 @@
+# Configure cluster-monitoring for console-operator
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: console-operator
+  namespace: openshift-console-operator
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      metricRelabelings:
+        - action: drop
+          regex: etcd_(debugging|disk|request|server).*
```

Comment:
does the console actually expose these metrics that they need to be dropped?

Reply:
No, I doubt it!  Copy paste on my part just to get it up and running. I'll remove this.

To be honest, the Console does not yet expose metrics at all. I need to add a `/metrics` endpoint to the `console` server, and a `prom-client` (and authenticate it).

Reply:
I'll cut the:

```
      metricRelabelings:
        - action: drop
          regex: etcd_(debugging|disk|request|server).*
          sourceLabels:
            - __name__
```